### PR TITLE
BatteryCare: Add version 0.9.36

### PR DESCRIPTION
**Pull request by fauzanelka**
- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

Add BatteryCare v0.9.36 app manifest

### CODE REVIEW

File: bucket/battery-care.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+    "version": "0.9.36",
+    "description": "BatteryCare is a software created to optimize the usage and performance of the modern laptop's battery. ",
```
Comment (rashil2000) [inline bucket/battery-care.json:3]:
```suggestion
    "description": "Tool to optimize the usage and performance of the modern laptop's battery. ",
```

**Comment by rashil2000** [comment]
Add autoupdate field (it is required). 

Where is the user data/config stored?

**Comment by fauzanelka** [comment]
> Add autoupdate field (it is required).
> 
> Where is the user data/config stored?

I forgot that, I see that BatteryCare loads user data in AppData directory. How do you set that?

File: bucket/battery-care.json (hunk 1 above)
Comment (fauzanelka) [reply bucket/battery-care.json:3]: Looks good, okay.

**Comment by rashil2000** [comment]
If it is stored in AppData, then no change required. Although portable applications shouldn't really store in AppData.

**Comment by fauzanelka** [comment]
What about the hash of the new version? The official URL only has one and it's the latest version.

**Comment by rashil2000** [comment]
If hash is not available, we can skip putting it in autoupdate.

**Comment by fauzanelka** [comment]
LGTM!

File: bucket/battery-care.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    "shortcuts": [
+        [
+            "BatteryCare.exe",
+            "BatteryCare"
+        ]
+    ],
+    "checkver": {
+        "url": "https://batterycare.net/en/download.html",
+        "regex": "Version ([\\d.]+)"
+    }
```
Comment (rashil2000) [inline bucket/battery-care.json:17]:
```suggestion
    },
    "autoupdate": {
        "url": "https://batterycare.net/files/BatteryCarePortable.zip"
    }
```

**Pull request merged**